src/version.ts

```diff
@@ -1 +1 @@
-export const VERSION = '1.7.0';
+export const VERSION = '1.8.0';
```

Comment:
The version was bumped to `1.8.0` in the last PR and has not been released yet.  We forgot to update the version here in that PR.